_tools/sycamore.md

```diff
@@ -11,23 +11,23 @@ has_children: false
 
 To get started, visit the [Sycamore documentation](https://sycamore.readthedocs.io/en/stable/sycamore/get_started.html).
 
-# Sycamore ETL pipeline structure
+## Sycamore ETL pipeline structure
 
 A Sycamore extract, transform, load (ETL) pipeline applies a series of transformations to a [DocSet](https://sycamore.readthedocs.io/en/stable/sycamore/get_started/concepts.html#docsets), which is a collection of documents and their constituent elements (for example, tables, blocks of text, or headers). At the end of the pipeline, the DocSet is loaded into OpenSearch vector and keyword indexes.
 
 A typical pipeline for preparing unstructured data for vector or hybrid search in OpenSearch consists of the following steps:
 
 * Read documents into a [DocSet](https://sycamore.readthedocs.io/en/stable/sycamore/get_started/concepts.html#docsets).
 * [Partition documents](https://sycamore.readthedocs.io/en/stable/sycamore/transforms/partition.html) into structured JSON elements.
-* Extract metadata, filter, and clean data using [transforms](https://sycamore.readthedocs.io/en/stable/sycamore/APIs/docset.html).
+* Extract metadata, filter and clean data using [transforms](https://sycamore.readthedocs.io/en/stable/sycamore/APIs/docset.html).
```

Comment:
Either "Extract metadata and filter and clean data using transforms" or "Extract metadata and filter and clean the source data using transforms"